python_modules/dagster/dagster/_core/execution/plan/inputs.py

```diff
@@ -123,7 +123,7 @@ def compute_version(
 
 
 @whitelist_for_serdes(storage_field_names={"node_handle": "solid_handle"})
-class FromSourceAsset(
+class FromLoadableAsset(
```

Comment:
Probably safest to use the serdes bit to keep the old persisted name. I don't recall the exact scenarios we load / save the execution plan snapshot across versions. Low cost to play it safe imo.